src/core_plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -66,16 +66,27 @@ uiModules
     location: 'Visualization Editor'
   });
 
+  // Retrieve the resolved SavedVis instance.
   const savedVis = $route.current.locals.savedVis;
 
+  // Instance of vis.js.
   const vis = savedVis.vis;
+
+  // Clone the _vis instance.
   const editableVis = vis.createEditableVis();
+
+  // NOTE: Why is this instance method being overwritten?
```

Comment:
The `editableVis` is intended to be a mirror of the vis object, and as such it needs to go through the same lifecycle methods that vis does (in this case `#requesting()`). I'm not certain what benefits from the requesting method being called on the editable vis though...